shared/configs.ts

```diff
@@ -48,17 +48,32 @@ export function matchFile(
   }
 
   configs.forEach((config, index) => {
+    const isFileGlobalIgnored = configArray.isFileIgnored(filepath)
+    let isFileIgnoredInCurrConfig = false
+    if (config.ignores) {
+      const ignoreConfigArray = buildConfigArray([{
+        index: config.index,
+        // only include ignores because of how ConfigArray works internally:
+        // isFileIgnored only works when only `ignore` exists
+        // (https://github.com/eslint/rewrite/blob/config-array-v0.19.1/packages/config-array/src/config-array.js#L726)
+        ignores: config.ignores ?? [],
```

Comment:
@antfu @comp615 Hi guys, I just updated the PR again here in this commit

On a special note, here in particular, we need to check the current config as well to make sure the file is not ignored in current config. See inline code comment for more details